packages/skeleton-svelte/src/lib/components/Progress/Progress.svelte

```diff
@@ -0,0 +1,49 @@
+<script lang="ts">
+	import type { ProgressProps } from './types.js';
+
+	let {
+		value,
+		min = 0,
+		max = 100,
+		// Root
+		base = 'overflow-x-hidden bg-surface-200-800',
+		height = 'h-2',
+		rounded = 'rounded',
+		classes = '',
+		// Indicator
+		indicatorBase = 'bg-primary-500 h-full',
```

Comment:
same here for the background classes, extract it to it's own prop.
I would leave the height in base since there no other values in base and users will not lose anything if they decide to override it.

Reply:
Done.